demos/multichannel_demo/yolo_v3/README.md

```diff
@@ -0,0 +1,129 @@
+# Multi-Channel Yolo v3 C++ Demo
+
+This demo provides an inference pipeline for multi-channel yolo v3. The demo uses Yolo v3 Object Detection network. You can follow [this](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_YOLO_From_Tensorflow.html) page convert the YOLO V3 and tiny YOLO V3 into IR model and execute the this demo with converted IR model.
+
+> **NOTES**:
+> This demo is default for yolo v3 model, if you want to run with tiny yolo v3 model, please patch the `anchors` vector. 
```

Comment:
is it possible to deduce from input what kind of YOLO model supplied and choose correct anchors at run time?

Reply:
I have removed this part since the anchor size is included in the model.